ttnn/cpp/ttnn/tensor/tensor_impl.cpp

```diff
@@ -755,16 +755,16 @@ Tensor to_device(const Tensor& tensor, Device* target_device, const MemoryConfig
     TT_FATAL(target_device != nullptr, "Need target device in order to move tensor to device!");
     TT_FATAL(tensor.is_allocated(), "Need data to exist in order to move it to device");
 
+    auto legacy_shape = tensor.get_shape();
```

Comment:
This actually returns ttnn::Shape, which we call Shape, and not LegacyShape. Lets tweak names here a bit